microsoft-edge/progressive-web-apps-chromium/how-to/microsoft-store.md

```diff
@@ -116,6 +116,20 @@ Use this feature to measure distinct traffic from your Store-installed PWA.  Bas
 This feature was first introduced in Microsoft Edge version 91, and the DOM API was introduced in Microsoft Edge version 93.
 
 
+<!-- ====================================================================== -->
+## Created a unified experience across multiple domains
+
+A Web App Manifest is tied to a single domain, but some PWAs use locale-specific domains for their customers in specific regions of the globe. When visiting the PWA in a web browser, customers are seamlessly transitioned from the principal domain (e.g., contoso.com) to a locale-specific one (e.g., contoso.co.ke) because the redirect happens during initial load of that website. Customers who install the PWA from Edge would therefor install it from the locale-specific domain and all future launches of that PWA will go directly to that domain instead of the principal one.
+
+Store-installed PWAs, however, have a hard-coded start URL that is pointed at the principal domain. When the PWA is launched, it navigates to the principal domain and then a customer may (as necessary) be redirected to their locale-specific domain. If that redireciton occurs, the navgation is considered "out of scope" and the app UI will display the URL and title at the top of the web page. This is a security feature to ensure users are aware that they have left the context of the app. That makes sense when users are loading a page from another website in the context of the PWA, but can be a little jarring when users are moving between domains that are all part of the same app.
```

Comment:
```suggestion
Store-installed PWAs, however, have a hard-coded start URL that is pointed at the principal domain. When the PWA is launched, it navigates to the principal domain and then a customer may (as necessary) be redirected to their locale-specific domain. If that redirection occurs, the navigation is considered "out of scope" and the app UI will display the URL and title at the top of the web page. This is a security feature to ensure users are aware that they have left the context of the app. That makes sense when users are loading a page from another website in the context of the PWA, but can be a little jarring when users are moving between domains that are all part of the same app.
```
I plan to make such fixes in a Writer/Editor pass.  quick note for now

Reply:
Changed app to website in the last sentence.

Reply:
@aarongustafson - can we add a screenshot of the security UI that displays when the user is redirected. That will help land the before and after picture to highlight the change that we're documenting in this PR.